misc/scripts/check_mixed_eol.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+cd `dirname "$0"`/../..
+
+exitcode=0
+
+check_eol()
+{
+    if [[ $1 == *.ani  || \
```

Comment:
I use
```sh
case "$file" in
*.awk | *.bib | *.bkl | *.c | *.cpp | *.cs | *.css | *.db33 | *.ddlmaker | *.dia | *.dox | *.doxy | *.h | *.hpp | *.java | *.js | *.json | *.lex | *.make | *.md | *.php | *.pl | *.pm | *.po | *.py | *.rst | *.sh | *.sql | *.tex | *.txt | *.xml | *.xrc | *.xslt | *.yaml | *.yml )
    expected_eoltype=unix
    ;;

*.bas | *.bat | *.cmd | *.csproj | *.iss | *.rc | *.rc2 | *.sln | *.vb | *.vbproj | *.vbs | *.vcproj | *.vcxproj | *.xaml )
    expected_eoltype=dos
    ;;

* )
    expected_eoltype=
    ;;
esac
```
not sure if this is much more readable, but personally I prefer this to `if`.

Reply:
Yes, this is more readable. I updated the script to use this.